coveralls.json

```diff
@@ -3,7 +3,8 @@
     "minimum_coverage": 87
   },
   "skip_files": [
-    "test/support"
+    "test/support",
+    "lib/course_planner/notifier.ex"
```

Comment:
please remove this from the exceptions, we should know that this file is not being covered when we see the coveralls result, if we can't/won't cover this will be an informed decision